portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase

```diff
@@ -461,7 +519,7 @@ definition {
 		}
 
 		task ("Choose a rating star for the DM") {
-			Click(locator1 = "Icon#INFO");
+			DMDocument.expandInfo();
```

Comment:
Hey @calvinctvo, Could you make this change in this [line](https://github.com/liferay-ac/liferay-portal/pull/60/files#diff-4e099d0c407cd79e5425318cf2e48b838a16c5165830cfaf7543d34ef84ec6deR568) too?

Reply:
Hey @ViniLopes20 , I'm not sure which line are you referring to?

Reply:
Sorry man, line 568